main.c

```diff
@@ -116,32 +134,18 @@ static void __interrupt() interrupt_handler() {
 // This is called from within can_handle_interrupt()
 static void can_msg_handler(const can_msg_t *msg) {
     uint16_t msg_type = get_message_type(msg);
-    int cmd_type = -1;
     switch (msg_type) {
-        case MSG_GENERAL_CMD:
-            // toggle ON_OFF to turn off module properly
-            cmd_type = get_general_cmd_type(msg);
-            if (cmd_type == BUS_DOWN_WARNING) {
-                LATC2 = 1;
-                __delay_ms(300);
-                LATC2 = 0;
-                __delay_ms(300);
```

Comment:
I take it we can't tell the new GPS to turn off?

Reply:
In the old module's datasheet, there is a section saying that the module needs to be properly shut down before power off. I didn't find anything like that in the new module's datasheet so I removed it.